ern-api-gen/src/apigen.js

```diff
@@ -132,9 +130,11 @@ async function checkValid (message: string) {
   }
   return pkg
 }
+
 async function readPackage () {
   return fileUtils.readJSON(`${process.cwd()}/${PKG_FILE}`)
```

Comment:
Same remark as previous, please use `path.join` here (not a change you did part of this PR, but can be updated part of this PR)